test/general/buffer_wrapper.pass.cpp

```diff
@@ -0,0 +1,75 @@
+#include "support/test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+#include _PSTL_TEST_HEADER(algorithm)
+#include _PSTL_TEST_HEADER(numeric)
+#include _PSTL_TEST_HEADER(iterator)
+
+#include "support/utils.h"
+#include <vector>
+
+
+//This macro is required for the tests to work correctly in CI with tbb-backend.
+#if TEST_DPCPP_BACKEND_PRESENT
+struct test_buffer_wrapper
+{
+    template <typename Iterator1, typename Iterator2>
+    void operator()(const Iterator1& begin1, const Iterator2& end1, std::size_t size)
+    {
+        std::fill(oneapi::dpl::execution::dpcpp_default,begin1,end1,1);
+        auto begin_host1 = TestUtils::get_host_pointer(begin1);
+        auto end_host1 = TestUtils::get_host_pointer(end1);
+
+        for(auto it = begin_host1; it!= end_host1; it=it+1)

```

Comment:
please, add spaces before and after `!=`, `=`, `+`

Reply:
Done